front/src/modules/pathfinding/hooks/usePathfinding.ts

```diff
@@ -205,6 +269,7 @@ const usePathfinding = ({
 
       if (!pathfindingInput) {
         setIsMissingParam();
+        await fetchOperationalPoints(pathSteps.filter((step): step is PathStep => step !== null));
```

Comment:
Why do we need the `step is PathStep`?

I think this is dangerous in case the type of `pathSteps` is changed to something else.